coil-gif/src/main/java/coil3/gif/imageRequests.kt

```diff
@@ -17,7 +19,11 @@ import coil3.request.Options
  * @see AnimatedImageDrawable.setRepeatCount
  */
 fun ImageRequest.Builder.repeatCount(repeatCount: Int) = apply {
-    require(repeatCount >= REPEAT_INFINITE) { "Invalid repeatCount: $repeatCount" }
+    if (SDK_INT >= 28) {
+        require(repeatCount >= ENCODED_LOOP_COUNT) { "Invalid repeatCount: $repeatCount" }
+    } else {
+        require(repeatCount >= REPEAT_INFINITE) { "Invalid repeatCount: $repeatCount" }
+    }
     extras[repeatCountKey] = repeatCount
 }
```

Comment:
In this part, the lint was broken because the version branch for `ENCODED_LOOP_COUNT` was not given, but this was fixed.